docs/library/sys.rst

```diff
@@ -15,6 +15,21 @@ Functions
    function raise as `SystemExit` exception. If an argument is given, its
    value given as an argument to `SystemExit`.
 
+.. function:: print_exception(exc, file=sys.stdout, /)
+
+   Print exception with a traceback to a file-like object *file* (or
+   `sys.stdout` by default).
+
+   .. admonition:: Difference to CPython
+      :class: attention
+
+      This is simplified version of a function which appears in the
+      ``traceback`` module in CPython. Unlike ``traceback.print_exception()``,
+      this function takes just exception value instead of exception type,
+      exception value, and traceback object; *file* argument should be
+      positional; further arguments are not supported. CPython-compatible
+      ``traceback`` module can be found in `micropython-lib`.
```

Comment:
I grabbed this description from the Micropython documentation and I'm not sure about the last sentence. What is the Circuitpython equivalent to 'micropython-lib'?

Reply:
Probably the CircuitPython bundle. We have separate repos for CPython compatible modules.

Reply:
Well I found the Micropython-lib reference and it looks like it adds additional methods to the library for cpython compatibility. I think it would be difficult to use with Circuitpython so unless someone thinks there's a better choice, I'm going to just drop that last sentence.